workers/go/activities/activities.go

```diff
@@ -1,7 +1,90 @@
 package activities
```

Comment:
I think we didn't organize this well when omes was first developed. I think activities and workflows that work together for a purpose should be together in a package. In other words, packages should be by purpose of thing (e.g. `throughputstress` package) not by type like activity or workflow.

Reply:
Yeah, agreed. I didn't change it just to avoid changing too much, but, I might as well do it now.